### DISCUSSION TITLE
breaking change?

### DISCUSSION THREAD

**Discussion by Willem-J-an** [rmagatti/auto-session#326] [Q&A] [answered]
With below config I was always using the workflow as follows:
If I want to restore my session, I do SessionSave, then close and open nvim with:
`nvim`
If I don't want to restore my session I would run `nvim .`

Since a recent update it seems `nvim .` now also restores the session. I don't see this in the breaking change issue, am I missing something? :)
Thanks!

```
{
                auto_save_enabled = true,
                auto_session_create_enabled = false,
                auto_restore_enabled = true,
                auto_session_use_git_branch = false,
                bypass_session_save_file_types = { "neo-tree" },
                pre_save_cmds = {
                    function()
                        require("neo-tree.sources.manager").close_all()
                    end,
                },
            }
```

**Comment by cameronr** [comment, marked as answer]
Thanks for the report. This was a recent change I made and probably should've been marked as a breaking change. You can disable the behavior by setting 'args_allow_single_directory = false'. More info here:

https://github.com/rmagatti/auto-session#argument-handling

**Comment by rmagatti** [comment]
@cameronr let's add this to https://github.com/rmagatti/auto-session/issues/186?
